blocks/tab-navigation/tab-navigation.js

```diff
@@ -0,0 +1,13 @@
+import { getMetadata } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
```

Comment:
Since querySelectorAll returns a NodeList, you don't need to check for its existence as it will always return a NodeList even if it's empty. Instead, you can check the length of the NodeList to see if there are any links.

You also don't need to spread the NodeList into an array with [...links] because NodeList objects are iterable, so you can directly use forEach on it. This can improve performance, especially if there are a large number of links.

```
import { getMetadata } from '../../scripts/lib-franklin.js';

export default function decorate(block) {
  const links = block.querySelectorAll('a');
  if (!links.length) return;
  
  const actualPage = getMetadata('og:url').split('/').slice(-1)[0].toLowerCase();
  
  links.forEach(a => {
    if (a.getAttribute('href').toLowerCase().includes(actualPage)) {
      a.classList.add('active');
    }
  });
}
```